### DISCUSSION TITLE
Phone display

### DISCUSSION THREAD

**Discussion by alexbrowng** [directus/directus#8540] [Feature Requests]
Create a display for fields containing a phone number that has an icon in the collection view that includes a "tel" link.

**Comment by benhaynes** [comment]
I like the idea for the email default values... but maybe we could combine all of this into the main input with a dropdown for "Click Action"? This seems core enough to go that route. @rijkvanzanten do you agree?

* External Link
* Email
* Telephone

**Reply by rijkvanzanten** [reply]
Yeah that makes the most sense to me. Have a single display that treats the value as a link, and allow the user to control the "protocol" (either http/tel/mailto/ or any app specific deep link)

**Reply by alexbrowng** [reply]
Yes, that seems more effective, not only for protocols, but also routes, such as opening a whatsapp chat.

**Reply by jugrajsingh** [reply]
Just like a Google sheets cell behaves.
It should automatically pickup the format from default or custom provided logic.

**Comment by anlexN** [comment]
good idea

**Comment by themegabyte** [comment]
Gentlemen, My apologies if this is a redundant question, is this available to be used? I spent some time looking through the admin App but I cannot seem to find a "Link" display type that would help me store the display information for this field.

**Comment by Dominic-Marcelino** [comment]
Just saw this discussion. Not in core, but we've build an extension for this:
https://github.com/utomic-media/directus-extension-field-actions

Allows to 
- link the value itself (in displays + readonly interfaces)
- adding a link-button (http / email / tel) 
- adding a copy button

**Comment by br41nslug** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
